### ISSUE TITLE
How to compile the css inject html

### ISSUE THREAD

**Issue by a13821190779** [egoist/rollup-plugin-postcss#50]
my rollup.confige.js is below
```javascript
const path = require('path')
const json = require('rollup-plugin-json')

const commonjs = require('rollup-plugin-commonjs')
const resolve = require('rollup-plugin-node-resolve')
// babel
const babel = require('rollup-plugin-babel')
// env
const replace = require('rollup-plugin-replace')

const uglify = require('rollup-plugin-uglify')

// css
const sass = require('node-sass')
const postcss = require('rollup-plugin-postcss')
const cssnano = require('cssnano')
const autoprefixer = require('autoprefixer')

const sassPreprocessor = (content, id) => new Promise((resolve) => {
    const result = sass.renderSync({ file: id })
    resolve({ code: result.css.toString() })
})

export default {
    name: 'fox',
    input: path.resolve(__dirname, '../src/index.js'),
    output: {
        file: path.resolve(__dirname, '../dist/bundle.js'),
        format: 'umd'
    },
    plugins: [
        json(),

        postcss({
            extract: true,
            sourceMap: true,
            extensions: ['.scss'],
            preprocessor: sassPreprocessor, // Pre-process all imports with Sass
            plugins: [
                autoprefixer(), 
                // cssnano()
            ]
        }),
        resolve({
            jsnext: true,
            main: true,
            browser: true,
            extensions: ['.js', '.json']
        }),
        commonjs({

        }),
        babel({
            exclude: 'node_modules/**'
        }),
        replace({
            exclude: 'node_modules/**',
            ENV: JSON.stringify(process.env.NODE_ENV || 'development')
        }),
        (process.env.NODE_ENV === 'production' && uglify())
    ],
    sourcemap: true
}
```

If anyone knows, please tell me, please
thanks very much

**Comment by egoist** [comment]
hmm I didn't get it. Can you rephrase a bit?

**Comment by a13821190779** [comment]
I use sass, and use rollup-plugin-postcss to compile it
it's configuration options extract is boolean, css was injected when tract is false or do not exist, css was  extracted when tract is true
but now, I need extract css and inject it into html with link tag in productioon mode, and inject it into html with style tag when development mode.
now I don't  know what to do.
My English is poor, sorry

**Comment by egoist** [comment]
this is actually not related to this plugin, but you can just different .html file in different mode
